packages/aws-cdk/lib/api/toolkit-info.ts

```diff
@@ -73,6 +76,82 @@ export class ToolkitInfo {
     return { filename, key, changed: true };
   }
 
+  /**
+   * Prepare an ECR repository for uploading to using Docker
+   */
+  public async prepareEcrRepository(id: string, imageTag: string): Promise<EcrRepositoryInfo> {
```

Comment:
This method probably should log some stuff out to `debug` at the very least. So we can tell what we're waiting for...